tests/fast-integration/image/Dockerfile

```diff
@@ -7,18 +7,12 @@ ENV FIT_MAKE_TARGET=ci-skr
 VOLUME /KUBECONFIG
 VOLUME /EMS
 
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    build-essential \
-    ca-certificates \
-    curl \
+RUN apk --no-cache add \
+    build-base \
     git \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN curl -fLSs https://deb.nodesource.com/setup_14.x | bash -
-RUN apt-get -y --no-install-recommends install nodejs
+    nodejs=16.14.2-r0
```

Comment:
I think it is safer to use the same nodejs version on both Prow and Spinnaker 🤔

Reply:
might makes sense. On the other hand wouldn't it be better to bump both to the latest LTS version (16.x) ?

Reply:
Also fine for me :) If it works, then I don't see a reason for not doing it :)